protocol/messenger_communities.go

```diff
@@ -3096,9 +3102,30 @@ func (m *Messenger) sendSharedAddressToControlNode(receiver *ecdsa.PublicKey, co
 
 	requestToJoin, err := m.communitiesManager.GetCommunityRequestToJoinWithRevealedAddresses(pk, community.ID())
 	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, communities.ErrRevealedAccountsAbsent
+		}
 		return nil, err
 	}
 
+	if len(requestToJoin.RevealedAccounts) == 0 {
+		return nil, communities.ErrRevealedAccountsAbsent
+	}
+
+	// check if at least one account is signed
+	// old community users can not keep locally the signature of their revealed accounts in the DB
+	revealedAccountSigned := false
+	for _, account := range requestToJoin.RevealedAccounts {
+		revealedAccountSigned = len(account.Signature) > 0
+		if revealedAccountSigned {
+			break
+		}
+	}
+
+	if !revealedAccountSigned {
+		return nil, communities.ErrNoRevealedAccountsSignature
+	}
```

Comment:
it would be nice to include this ticket to RC 0.15.x or we need to move this selected code to RC 0.15.x

Reply:
The rest of the code doesn't seem to introduce regressions, so it should be ok to put in the release.